File: How old will I be in 2099.rb
def calculate_age(year_of_birth, current_year)
  if year_of_birth > current_year
    age = year_of_birth - current_year
    if age == 1
      return "You will be born in #{age} year."
    else
      return "You will be born in #{age} years."
    end
  elsif year_of_birth < current_year
      age = current_year - year_of_birth
      if age == 1
        return "You are #{age} year old."
      else
        return "You are #{age} years old."
      end
  else
    return "You were born this very year!"
  end
end

File: How old will I be in 2099.js
function calculateAge(birthYear, askedYear) {
  switch (true) {
    case birthYear === askedYear:
      return "You were born this very year!";
      break;
    case birthYear < askedYear:
      age = askedYear - birthYear;
      if (age === 1) {
        return `You are ${age} year old.`;
      } else {
        return `You are ${age} years old.`;
      }
      break;
    case birthYear > askedYear:
      age = birthYear - askedYear;
      if (age === 1) {
        return `You will be born in ${age} year.`;
      } else {
        return `You will be born in ${age} years.`;
      }
      break;
    default:
      "Unable to calculate. Check with mom.";
      break;
  }
}